package.json

```diff
@@ -56,33 +55,33 @@
     "moment": "^2.29.1",
     "morgan": "^1.10.0",
     "nocache": "^3.0.1",
-    "node-sass-middleware": "0.11.0",
```

Comment:
no longer have prod dependency on node-sass-middleware and node-sass which often have lots of vulnerabilities associated with them.